doc/book/replication/repl_leader_elect.rst

```diff
@@ -69,10 +69,15 @@ Eventually, a leader is elected.
 
 All the non-leader nodes are called *followers*. The nodes that start a new
 election round are called *candidates*. The elected leader sends heartbeats to
-the non-leader nodes to let them know it is alive. So if there are no heartbeats
-for a period set by the :ref:`replication_timeout <cfg_replication-replication_timeout>`
-option, a new election starts. Terms and votes are persisted by
-each instance in order to preserve certain Raft guarantees.
+the non-leader nodes to let them know it is alive.
+
+In case there are no heartbeats for a period set by the :ref:`replication_timeout <cfg_replication-replication_timeout>` option,
+a non-leader node starts a new election if the following conditions are met:
+
+*   It has a quorum of connections to other cluster members.
+*   None of these cluster members can see the leader node.
```

Comment:
I don't know if it's worth mentioning or not. You decide:
The other cluster members think that they see the leader node, if they have heard from it at least once during the `replication_death_timeout` period AND there were no replication errors (the connection is not broken due to timeout or due to an error).